iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/vector.inl

```diff
@@ -237,16 +244,20 @@ inline bool vector<T, Capacity>::resize(const uint64_t count, const Targs&... ar
     {
         while (count != m_size)
```

Comment:
Wait, why does this work?? This is an infinite loop since count is never incremented?

Reply:
`m_size` is incremented in `emplace_back`.